concordium-std/src/types.rs

```diff
@@ -643,6 +646,18 @@ pub struct QueryAccountBalanceError;
 #[derive(Debug, Copy, Clone, PartialEq, Eq)]
 pub struct QueryContractBalanceError;
 
+/// Error for querying account's public keys.
+/// No account found for the provided account address.
+#[derive(Debug, Copy, Clone, PartialEq, Eq)]
+pub struct QueryAccountPublicKeysError;
+
+/// Error for checking an account signature.
+#[derive(Debug, Copy, Clone, PartialEq, Eq)]
+pub enum CheckAccountSignatureError {
+    MissingAccount,
+    MalformedData,
```

Comment:
Please add documentation for these variants. `MissingAccount` is somewhat obvious, but I don't think `MalformedData` is

Reply:
Added